src/redux/messages/messageReducer.js

```diff
@@ -0,0 +1,19 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+export const fetchMessages = createAsyncThunk('rail-react/messages', async () => {
+  const response = await fetch('http://localhost:3000/api/v1/messages');
```

Comment:
- [ ] Kindly double check the route since it is throwing an error. I think instead of messages it should be greetings but even after that I wasn't getting any greating. Kindly double check the controller whether the data is coming and remember you need to show only 1 random greeting
![image](https://github.com/maaloum/hello_rails_fontend/assets/62705947/a1e64cbe-b0af-4991-8322-0f9a9fd2070e)